src/components/ProductList/ProductList.js

```diff
@@ -59,40 +60,46 @@ export const ProductList = () => {
                 dispatchHandler(
                   wishlistDispatch,
                   ACTION_TYPE.ADD_TO_WISHLIST,
-                  filterProduct
+                  filterProduct,
+                  toast.success("Added to wishlist")
                 )
               }
               removeFromWishlist={() =>
                 dispatchHandler(
                   wishlistDispatch,
                   ACTION_TYPE.REMOVE_FROM_WISHLIST,
-                  filterProduct
+                  filterProduct,
+                  toast.success("Removed from wishlist")
```

Comment:
I feel we dont need a toast message for removing from cart / wishlist when user is moving the product from cart to wishlist or vice versa